README.md

```diff
@@ -38,20 +38,27 @@ PRs should be posted _before_ class begins on Tuesday, January 2, 2024.
 ### Interview Practice Questions
 
 1. What are three (3) syntax difference between HTML and JSX?
-
+   //class vs className
+   //inline styles
+   //self-closing tags
 2. What is yarn? What file(s) are modified when you run the command yarn in your terminal?
-
+  //It is a popular package manager for Node.js applications.
+   //When I run it, it installs package.json file so that the source can be used and shared.
```

Comment:
Technically when you run yarn it uses the existing package.json to build the node modules folder with all the appropriate dependencies.